### PR TITLE
Email notification

### PR DESCRIPTION
**Pull request by nicozar95**
## Summary 
* Added email notification when a rent is created
* Sidekiq implementation to perform asyncronic mail send

## Trello
https://trello.com/c/obwTR6QP/11-mandar-email-al-usuario-cuando-se-crea-un-rent

### CODE REVIEW

File: app/workers/async_mailer_worker.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class AsyncMailerWorker
+  include Sidekiq::Worker
```
Comment (juanisierra) [inline app/workers/async_mailer_worker.rb:2]: Do you use this worker?

File: config/environments/development.rb [hunk 2]
```diff
@@ -51,8 +51,28 @@
...
+  :address => Rails.application.secrets.mailer_address,
+  :domain => Rails.application.secrets.mailer_domain,
+  :port => Rails.application.secrets.mailer_port,
+  :authentication => Rails.application.secrets.mailer_authentication
+}
+
+
+  #config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
+##  config.action_mailer.delivery_method = :smtp
```
Comment (juanisierra) [inline config/environments/development.rb]: Remove comments please

File: spec/fixtures/user_mailer/rent_finished_notification [hunk 3]
```diff
@@ -0,0 +1,3 @@
+UserMailer#rent_finished_notification
```
Comment (juanisierra) [inline spec/fixtures/user_mailer/rent_finished_notification]: Do you use this for something?

**Review by juanisierra — CHANGES_REQUESTED** [review]

File: app/workers/async_mailer_worker.rb (hunk 1 above)
Comment (nicozar95) [reply app/workers/async_mailer_worker.rb:2]: Yes

**Review by juanisierra — APPROVED** [review]

**Pull request merged**